.github/workflows/build.yml

```diff
@@ -19,4 +21,5 @@ jobs:
       uses: PreternaturalAI/github-action/preternatural-build@main
       with:
         xcode-version: '16'
-        configuration: 'debug'
\ No newline at end of file
+        platforms: 'all'
+        configuration: 'debug'
```

Comment:
@tahabebek this needs to be `configurations`

Reply:
fixed in https://github.com/PreternaturalAI/AI/pull/26